pkg/liqoctl/info/peer/network.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2019-2024 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package peer
+
+import (
+	"context"
+	"fmt"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	liqov1beta1 "github.com/liqotech/liqo/apis/core/v1beta1"
+	networkingv1beta1 "github.com/liqotech/liqo/apis/networking/v1beta1"
+	"github.com/liqotech/liqo/pkg/liqoctl/info"
+	"github.com/liqotech/liqo/pkg/liqoctl/info/common"
+	"github.com/liqotech/liqo/pkg/liqoctl/output"
+	"github.com/liqotech/liqo/pkg/utils/getters"
+)
+
+// GatewayType represents the type of network gateway (Client or Server).
+type GatewayType string
+
+const (
+	// GatewayClientType indicates that the gateway acts as a client.
+	GatewayClientType GatewayType = "Client"
+	// GatewayServerType indicates that the gateway acts as a server.
+	GatewayServerType GatewayType = "Server"
+)
```

Comment:
We can recycle consts already defined at https://github.com/liqotech/liqo/blob/master/pkg/consts/externalnetwork.go#L30-L33

Reply:
I created new constants for consistency with `Provider` and `Consumer`, which have the first letter capitalized